### DISCUSSION TITLE
New Ideas and Exploring OpenSSP Resources

### DISCUSSION THREAD

**Discussion by kiche-allan** [OpenSPP/documentation#10] [Ideas]
Hello Guys, so @jeremi  had earlier on shared via email onn suggested areas - such as global outreach, community building, partnerships, localization, and accessibility improvements and demonstrating how AI can be used to help developers adapt OpenSPP, explore the OpenSPP resources, share your project preferences or any new ideas

So this discussion is just a backtrack to this incase anyone has some suggestions to anything after going through the resources shared and even the documentation, this could be a good board to atleast discuss anything we find interesting....your thoughts?

**Comment by kiche-allan** [comment]
@jeremi have some few questions...
1. In terms of localization, what are your target languages
2. Looks like cloning and running the doc on local pc is only available on macos and ubuntu environments

**Reply by jeremi** [reply]
1. The main languages (English, Arabic, Spanish, French) and the one where we have people using the software. I started experimenting with using LLMs to help translate faster even for not so popular languages like Lao. It is working fairly well, but would need to be industrialized.
2. yes, but you can access it from: https://docs.openspp.org/

**Reply by Chifrica** [reply]
Right now I think we can utilize the presence of GitHub copilot to help developers to be effective and productive in their works.
